syntax = "proto3";
package com.example.grpc;

message GetFileRequest {
    string file_uuid = 1;
}

message CreateFileRequest {
    string file_uuid = 1;
    string data = 2;
}

message Response {
    string file_uuid = 1;
    string data = 2;
}

service LogFileService {
    rpc get_file(GetFileRequest) returns (Response);
    rpc create_file(CreateFileRequest) returns (Response);
}
